questions.json

```
@@ -118,5 +118,45 @@
     "explanation": "fork() 수행시 부모 프로세스의 PCB를 복사하는 것은 맞지만, PID와 PPID 등 고유한 영역은 새롭게 생성됩니다",
     "sources": [],
     "tags": ["OS"]
+  },
+  {
+    "question": "다음 중 트랜잭션의 특징에 대한 설명중 틀린 것을 고르세요.",
+    "choices": ["원자성(Atomicity) : 트랜잭션 안의 모든 작업은 전부 성공하거나 전부 실패해야 한다.", "일관성(Consistency) : 트랜잭션 이후에도 제약 조건을 유지해야 한다.", "격리성(Isolation) : 모든 트랜잭션은 서로 영향을 주지 않아야 한다.", "지속성(Durability) : 트랜잭션 이후에도 DB 상태는 유지되어야 한다."],
+    "correct": "지속성(Durability) : 트랜잭션 이후에도 DB 상태는 유지되어야 한다.",
+    "explanation": "지속성은 트랜잭션이 성공하면 DB에 반영되어야 함을 의미합니다. 이는 중간에 시스템 오류로 종료되더라도 반영되어야 합니다. 키워드 : binlog",
+    "sources": [],
+    "tags": ["DB"]
+  },
+  {
+    "question": "다음 중 트랜잭션 격리 수준에 대한 설명으로 틀린 것을 고르세요.",
+    "choices": ["READ UNCOMMITED : 커밋되지 않은 데이터라도 읽을 수 있는 수준", "READ COMMITED : 커밋된 데이터만 읽을 수 있는 수준. Non-Repeatable Read 문제가 발생한다.", "REPEATABLE READ : 같은 트랜잭션 내에서 같은 레코드를 조회했을 때 변경하지 않았다면 멱등성을 보장받을 수 있는 수준. Phantom Read 문제가 발생한다.", "SERIALIZABLE : 트랜잭션을 순차적으로 진행시킨다. MVCC를 사용해서 Lock 대기를 줄일 수 있다."],
```

```suggestion
    "choices": ["READ UNCOMMITED : 커밋되지 않은 데이터라도 읽을 수 있는 수준", "READ COMMITED : 커밋된 데이터만 읽을 수 있는 수준. Non-Repeatable Read 문제가 발생한다.", "REPEATABLE READ : 같은 트랜잭션 내에서 같은 레코드를 조회했을 때 변경하지 않았다면 멱등성을 보장받을 수 있는 수준. Phantom Read 문제가 발생한다.", "SERIALIZABLE : 트랜잭션이 시작되면 완료될 때까지 외부의 읽기 접근을 차단(Lock)하여 일관된 읽기를 보장한다. MVCC를 사용해서 Lock 대기를 줄일 수 있다."],
```

다른 트랜잭션 레벨도 순차적으로는 진행되지만 일관된 읽기를 보장하지는 못한다는 점에서 문맥을 약간 수정해보았습니다.